src/dotnet-grpc/Commands/CommandBase.cs

```diff
@@ -442,19 +442,7 @@ private static bool IsStreamContentIdentical(Stream remote, Stream local)
 
     private static byte[] GetHash(Stream stream)
     {
-        SHA256 algorithm;
-        try
-        {
-            algorithm = SHA256.Create();
-        }
-        catch (TargetInvocationException)
-        {
-            // SHA256.Create is documented to throw this exception on FIPS-compliant machines. See
-            // https://msdn.microsoft.com/en-us/library/z08hz7ad Fall back to a FIPS-compliant SHA256 algorithm.
-            algorithm = new SHA256CryptoServiceProvider();
-        }
-
-        using (algorithm)
+        using (var algorithm = SHA256.Create())
         {
             return algorithm.ComputeHash(stream);
```

Comment:
`return SHA256.HashData(stream);`

Reply:
Good idea, but `SHA256.HashData` doesn't support stream: https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.sha256.hashdata?view=net-6.0

Reply:
Oh sad, it's 7.0 https://apisof.net/catalog/e98725f1-0fcc-2eca-c31d-23e541ddcef8